README.md

```diff
@@ -157,7 +159,9 @@ patched_versions:
 * `notes` \[String\] (optional): Internal notes regarding the vulnerability's inclusion in this database.
 
 [CVSSv2]: https://www.first.org/cvss/v2/guide
-[CVSSv3]: https://www.first.org/cvss/user-guide
+[CVSSv3]: https://www.first.org/cvss/v3.0/user-guide
+[CVSSv3]: https://www.first.org/cvss/v3.1/user-guide
+[CVSSv3]: https://www.first.org/cvss/v4.0/user-guide
```

Comment:
You forgot to update the `v3` part of the link name.

Reply:
fixed